src/components/ContentCard/ContentCard.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import "./contentCard.css";
+
+// TODO: support variant
+
+export interface ContentCardProps {
+  /**
+   * Title of ContentCard
+   */
+  title: string;
+
+  /**
+   * Description of ContentCard
+   */
+  description: string;
+
+  /**
+   * Link to full content referenced by ContentCard
+   */
+  hyperlink: string;
+
+  /**
+   * Relative path to image 
+   */
+  image?: string;
+
+  /**
+   * Alt text for image. Compulsory field if image field is present
+   */
+  imgAlt?: string;
+
+  /**
+   * Relative path to bgImage 
+   */
+  bgImage?: string;
+
+  drawBorder?: boolean;
```

Comment:
this attribute is never used, sama g ad description nya

Reply:
Woops ini lupa gw delete lel